crates/cargo-util/src/paths.rs

```diff
@@ -784,4 +785,16 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn normalize_path_works() {
```

Comment:
I am not sure why this test appears in the pull request.

Reply:
Oh I was testing what that function exactly did and the comments weren't too specific so I tried doing some simple tests, because it said on the docs of that function that was error prone I kept the tests, so at least those simple cases are sure to work